services/hanke-service/src/test/kotlin/fi/hel/haitaton/hanke/factory/HakemusUpdateRequestFactory.kt

```diff
@@ -94,6 +94,16 @@ object HakemusUpdateRequestFactory {
                 createCustomerWithContactsRequest(type, yhteystietoId, *hankekayttajaIds)
         )
 
+    fun JohtoselvityshakemusUpdateRequest.withContractorWithContactsRequest(
```

Comment:
Longer names don't always mean clearer code. There's only one thing `withContractor` can do. Making the name longer makes it harder to parse without any added benefit.

Besides, this sets the `contractorWithContacts` field, not a `contractorWithContactsRequest` field.